src/views/profile/profile-update-password/profile-update-password.ts

```diff
@@ -59,9 +59,8 @@ export class ProfileUpdatePassword extends Block {
   onSubmit(e) {
     const fields = ['oldPassword', 'newPassword'];
     serializer(e, fields);
-    let dataForm: any = {};
+    let dataForm = {};
```

Comment:
Думаю что это стоит заменить на 
```suggestion
    const dataForm = serializer(e, fields);
```